install.ps1

```diff
@@ -257,9 +266,9 @@ if (-not $noChecks.IsPresent) {
     }
 
     # Internet connectivity checks
-    Test-WebConnection "google.com"
-    Test-WebConnection "github.com"
-    Test-WebConnection "raw.githubusercontent.com"
+    Test-WebConnection 'microsoft.com'
```

Comment:
> I added Microsoft.com instead of Google because of how likely of each to be blocked. This is also for personal privacy reasons, this features adds nothing extra but on a windows machine is usually preferred. I chose not to do a more privacy respecting website because that might also be blocked in a weird corporate environment.

Testing `google.com` is one of most (if not the most) used internet connectivity check. I see no reason why a corporate environment would block it. This looks to me like a personal preference and not an improvement of the code, so I would prefer to keep it as it was. Other opinions @mandiant/flare-vm?

@CamoCatX can you please remove this change so that we can merge this PR?